OKTA-404918  .NET IDX SDK > Add readme with all the steps/env vars/config required to run E2Es (#96)
okta/okta-idx-dotnet: samples/samples-aspnet/embedded-auth-with-sdk/README.md

for Embedded Authentication](TBD)

[OIDC Web Application Setup Instructions]: https://developer.okta.com/authentication-guide/implementing-authentication/auth-code#1-setting-up-your-application

## Running UI automation tests

> UI testing project `embedded-auth-with-sdk.E2ETests` uses SpecFlow and Selenium with chrome driver. In order to run the tests several configuration values need to be defined in the `settings.json` or in the system environment. Although this is not required, sensitive data should be kept as environment variables.

Reviewer: ```suggestion
UI testing project `embedded-auth-with-sdk.E2ETests` uses SpecFlow and Selenium with chrome driver. In order to run the tests, several configuration values need to be defined in the `settings.json` or in the system environment. Although this is not required, sensitive data should be kept as environment variables.
```

### Environment variables
 * `DirectAuthWebSitePath` - local path to embedded-auth-with-sdk project: `<...>\okta-idx-dotnet\samples\samples-aspnet\embedded-auth-with-sdk\embedded-auth-with-sdk`.
 * `okta_testing_A18nApiKey`- Api key for A18N service.

>  * `okta_testing_FacebookMfaUserEmail` - pre-created Facebook user's name. The should be already registered as an Okta user and added to `MFA Required` or equivalent group.

Reviewer: ```suggestion
 * `okta_testing_FacebookMfaUserEmail` - pre-created Facebook user's name. The user should be already registered as an Okta user and added to the `MFA Required` or equivalent group.
```

 * `okta_testing_FacebookMfaUserPassword`- pre-created Facebook user's password.

>  * `okta_testing_FacebookUserEmail` - pre-created Facebook user's name, should not exist in Okta's People Directory.

Reviewer: ```suggestion
 * `okta_testing_FacebookUserEmail` - pre-created Facebook user's name. It should not exist in Okta's People Directory.
```

 * `okta_testing_FacebookUserPassword` - password for the pre-created Facebook user not existing in Okta.

>  * `okta_testing_GoogleUserEmail` - pre-created Google user name, should not exist in Okta's People Directory.

Reviewer: ```suggestion
 * `okta_testing_GoogleUserEmail` - pre-created Google user's name. It should not exist in Okta's People Directory.
```
